packages/components/molecules/f-rating/stories/RatingMultiStar.stories.js

```diff
@@ -25,7 +25,8 @@ export const RatingComponent = (args, { argTypes }) => ({
                     v-bind="$props"
                     :starRatingSize="rating.starRatingSize"
                     :starRating="rating.starRating"
-                    :reviewCount="rating.reviewCount" />
+                    :reviewCount="rating.reviewCount"
```

Comment:
Can we make all of these props kebab-cased, please?

```suggestion
                    :review-count="rating.reviewCount"
```